actors/builtin/miner/miner_test.go

```diff
@@ -608,6 +608,22 @@ func TestCommitments(t *testing.T) {
 		st.FeeDebt = big.Zero()
 		rt.ReplaceState(st)
 		rt.Reset()
+
+		// Try to precommit with an active consensus fault
+		st = getState(rt)
+		params := &miner.ReportConsensusFaultParams{
+			BlockHeader1:     nil,
+			BlockHeader2:     nil,
+			BlockHeaderExtra: nil,
+		}
+
+		actor.reportConsensusFault(rt, addr.TestAddress, params)
```

Comment:
Nit: push the params construction down inside reportConsensusFault so this isn't so noisy.